src/mmw/apps/geoprocessing_api/validation.py

```diff
@@ -69,3 +71,97 @@ def validate_uuid(uuid):
         return False
 
     return uuid == str(u)
+
+
+def validate_gwlfe_prepare(data):
+    area_of_interest = data.get('area_of_interest')
+    wkaoi = data.get('wkaoi')
+    huc = data.get('huc')
+    layer_overrides = data.get('layer_overrides', {})
+
+    if not check_gwlfe_only_one([area_of_interest, wkaoi, huc]):
+        error = ('Invalid parameter: One and only one type of area object'
+                 ' (area_of_interest, wkaoi, or huc) is allowed')
+        raise ValidationError(error)
+
+    not_valid_layers = check_layer_overrides_keys(layer_overrides)
+
+    if not_valid_layers:
+        error = create_layer_overrides_keys_not_valid(not_valid_layers)
+        raise ValidationError(error)
+
+    if ('__LAND__' in layer_overrides.keys() and
+            not check_land_layer_overrides(layer_overrides)):
+        error = 'Invalid __LAND__ param: Must be one of the NLCD values'
```

Comment:
Instead of saying "one of the NLCD values", consider specifying the allowed values in a list, as is done here:

https://github.com/WikiWatershed/model-my-watershed/blob/9b1aebada97f14480a3fdbab70491ab24e8d8993/src/mmw/apps/geoprocessing_api/schemas.py#L197